websites/V/Vukkybox/dist/metadata.json

```diff
@@ -4,12 +4,24 @@
     "name": "Mitchellonii",
     "id": "490664749315653642"
   },
+  "contributors": [
+    {
+      "name": "Vukky",
+      "id": "708333380525228082"
+    },
+    {
+      "name": "Skelly",
+      "id": "125644326037487616"
+    }
```

Comment:
Surely not 
![image](https://user-images.githubusercontent.com/64903135/147515399-63b9267b-3c83-4880-8ce1-07c06d8ce1f4.png)